### DISCUSSION TITLE
Private messages & comments

### DISCUSSION THREAD

**Discussion by alvarpb** [globaleaks/globaleaks-whistleblowing-software#3447] [General]
I would like to create private comments internally without being read by the informant. I have tried both the comments and the messages, but in both cases the informant can read them.

Thanks,

**Comment by evilaliv3** [comment]
Thank you for your question.

This feature is currently not available but some users have proposed it and we have a partial specification: https://github.com/globaleaks/GlobaLeaks/issues/2452

Please feel free to add your feedback about the current that would be helpful to developers interested in supporting the project.

**Reply by alvarpb** [reply]
Thanks a lot
